HubFramework.xcodeproj/project.pbxproj

```diff
@@ -1334,6 +1334,19 @@
 				GCC_DYNAMIC_NO_PIC = NO;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				SDKROOT = iphoneos;
+				WARNING_CFLAGS = (
```

Comment:
All warnings should be added via the [`spotify_os.xcconfig`](https://github.com/spotify/HubFramework/blob/master/spotify_os.xcconfig) otherwise we can’t override the settings internally.